lang/cs/Org.Apache.REEF.Utilities/AsyncUtils/LoggingHelper.cs

```diff
@@ -36,8 +37,10 @@ public static void LogAndIgnoreExceptionIfAny(this Task self, Logger logger, str
         {
             self.ContinueWith(t =>
             {
+                var message = string.IsNullOrWhiteSpace(msg) ? string.Empty : msg + " ";
```

Comment:
why the `+ " "` at the end? Can't we add the whitespace in the format string below?